test/index.test.ts

```diff
@@ -289,7 +297,7 @@ describe('Probot', () => {
       const redisConfig = {
         host: 'test'
       }
-      const probot = createProbot({ webhookPath: '/webhook', githubToken: 'faketoken', redisConfig})
+      probot = createProbot({ webhookPath: '/webhook', githubToken: 'faketoken', redisConfig })
 
       expect(probot.throttleOptions.Bottleneck).toBe(Bottleneck)
       expect(probot.throttleOptions.connection).toBeInstanceOf(Bottleneck.IORedisConnection)
```

Comment:
Property 'throttleOptions' is private and only accessible within class 'Probot'.
Property 'IORedisConnection' does not exist on type 'typeof import("bottleneck")'.